Mathlib/Data/Set/Prod.lean

```diff
@@ -310,6 +315,9 @@ theorem snd_image_prod {s : Set α} (hs : s.Nonempty) (t : Set β) : Prod.snd ''
     let ⟨x, x_in⟩ := hs
     ⟨(x, y), ⟨x_in, y_in⟩, rfl⟩
 
+lemma mapsTo_snd_prod {s : Set α} {t : Set β} : MapsTo (Prod.snd) (s ×ˢ t) t :=
```

Comment:
```suggestion
lemma mapsTo_snd_prod {s : Set α} {t : Set β} : MapsTo Prod.snd (s ×ˢ t) t :=
```

Reply:
I wonder if both this one and `snd_image_prod_subset` are really necessary...

Reply:
In such basic files, I think it's important to have a very thorough API, because it will be used in so many places. So I'd say both are worth keeping.

Reply:
In that case, maybe the proof of `snd_image_prod_subset` should be golfed with [.image_subset](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Function.html#Set.MapsTo.image_subset)

Reply:
The proofs of all these lemmas are already one-liners, I'm not sure there's anything to be gained from golfing further.